hw/ip/hmac/doc/programmers_guide.md

```diff
@@ -88,17 +88,22 @@ When SW doesn't know each instant at which a full message block is available, it
 
 The context that needs to be saved and restored is in the following registers: [`CFG`](registers.md#cfg), [`DIGEST_*`](registers.md#digest), and [`MSG_LENGTH_*`](registers.md#msg_length_lower).
 
+**An RTL bug has been found where the HW is unable to recover if a stop command is issued after the HASH has been processed, more details [here](https://github.com/lowRISC/opentitan/issues/24767). A SW [workaround has been found](https://github.com/lowRISC/opentitan/pull/24944) and implies to adapt the way to use this feature. The temporary changes are marked in italics and replace the previous "normal" operations, which are striked through.**
```

Comment:
Please use one sentence per line as recommended in the [corresponding style guide](https://opentitan.org/book/doc/contributing/style_guides/markdown_usage_style.html).

Reply:
I suggest rephrasing this to something like:

Due to an RTL bug, the hardware may not be able to easily recover from the `Stop` command (for more details, refer to [Issue #24767](https://github.com/lowRISC/opentitan/issues/24767)).
To avoid the hardware from entering this state, a stopgap software workaround solution should be used (see [PR #24944](https://github.com/lowRISC/opentitan/pull/24944) for details).
The example usage below includes this stopgap workaround solution marked in italics whereas the original guidance is crossed out.

Reply:
OK